README.md

```diff
@@ -1795,6 +1795,7 @@ The following extractors use this feature:
 * `key_query`: Passthrough the master m3u8 URL query to its HLS AES-128 decryption key URI if no value is provided, or else apply the query string given as `key_query=VALUE`. Note that this will have no effect if the key URI is provided via the `hls_key` extractor-arg. Does not apply to ffmpeg
 * `hls_key`: An HLS AES-128 key URI *or* key (as hex), and optionally the IV (as hex), in the form of `(URI|KEY)[,IV]`; e.g. `generic:hls_key=ABCDEF1234567980,0xFEDCBA0987654321`. Passing any of these values will force usage of the native HLS downloader and override the corresponding values found in the m3u8 playlist
 * `is_live`: Bypass live HLS detection and manually set `live_status` - a value of `false` will set `not_live`, any other value (or no value) will set `is_live`
+* `impersonate`: Targets to try and impersonate in the website request; e.g. `safari,chrome-110`. By default any available target will be used. Set to `false` to not use impersonation.
```

Comment:
This does the exact same thing as `--impersonate`, except that it's enabled by default

I think a better solution (from a UI perspective, I don't know about the code perspective) would be to just add a --no-impersonate option

Reply:
No it does not. Its only for the website request. Having `--no-impersonate` would be obsoleted by `--impersonate false` once impersonate handlers will support multiple targets

Reply:
```suggestion
* `impersonate`: Target(s) to try and impersonate with the initial webpage request; e.g. `safari,chrome-110`. By default any available target will be used. Use `false` to disable impersonation
```